core/controllers/pages_test.py

```diff
@@ -15,14 +15,149 @@
 """Tests for various static pages (like the About page)."""
 
 from core.tests import test_utils
+import feconf
 
 
-class NoninteractivePagesTests(test_utils.GenericTestBase):
+class AboutPageTest(test_utils.GenericTestBase):
+    """Test for About page."""
 
-    def test_about_page(self):
-        """Test the About page."""
-        response = self.get_html_response('/about')
+    def test_loading_about_page_loads_correctly(self):
+        response = self.testapp.get('/about')
+        self.assertEqual(response.status_int, 200)
         self.assertEqual(response.content_type, 'text/html')
         response.mustcontain(
-            'I18N_ABOUT_PAGE_CREDITS_TAB_HEADING',
-            'I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_5_LICENSE_HEADING')
+            'I18N_ABOUT_PAGE_CREDITS_TAB_HEADING')
+
+
+class SplashPageTest(test_utils.GenericTestBase):
+    """Test for splash page."""
+
+    def test_loading_splash_page_loads_correctly(self):
+        response = self.testapp.get('/splash')
+        self.assertEqual(response.status_int, 200)
+        self.assertEqual(response.content_type, 'text/html')
+        response.mustcontain('I18N_SPLASH_TITLE')
+
+
+    def test_loading_splash_page_with_c_value_loads_correctly(self):
```

Comment:
loads_correctly --> leads_to_redirect (or is_redirected)

Reply:
_(adding a comment to mark unapproved)_

Reply:
Done!...